chips/nrf52/src/uart.rs

```diff
@@ -121,6 +120,10 @@ impl UART {
         let regs = unsafe { &*self.regs };
         let tx = regs.event_endtx.get() != 0;
 
+        if regs.event_error.get() == 1 {
+            panic!("uart error event\r\n");
```

Comment:
Oops, I just used this for debugging when I thought it was an issue with the UART.
As the UART transmit the entire buffer I think it is safe to just clear to error and try to re-send the buffer again. Otherwise just return and wait for the next "message"?

Thoughts?